### PR TITLE
chore: Update to AtlasMap 2.0.2

### PR DESCRIPTION
**Pull request by igarashitm**
Includes: https://issues.redhat.com/browse/ENTESB-13739
Backporting: #8468

### CODE REVIEW

**Comment by igarashitm** [comment]
Big changes in the yarn.lock - that was generated from yarn install

**Comment by riccardo-forina** [comment]
> Big changes in the yarn.lock - that was generated from yarn install

Yes unfortunately it's normal. I haven't found a way to make this kind of tasks suck less, yet.

File: app/ui-react/packages/atlasmap-adapter/src/DataMapperAdapter.tsx [hunk 1]
```diff
@@ -83,9 +83,11 @@ export const DataMapperAdapter: React.FunctionComponent<IDataMapperAdapterProps>
       onMappingChange={onMappings}
     >
       <Atlasmap
-        showImportAtlasFileToolbarItem={false}
-        showExportAtlasFileToolbarItem={false}
-        showResetToolbarItem={false}
+        allowImport={false}
+        allowExport={false}
+        allowReset={false}
+        allowDelete={false}
```
Comment (riccardo-forina) [inline app/ui-react/packages/atlasmap-adapter/src/DataMapperAdapter.tsx:89]: @igarashitm I took the liberty to update your PR with these 2 options required to fully disable features we don't want in Syndesis.

Comment (igarashitm) [reply app/ui-react/packages/atlasmap-adapter/src/DataMapperAdapter.tsx:89]: thx!

**Pull request merged**